app/views/facilities/show.html.haml

```diff
@@ -24,12 +26,14 @@
   = form_for @order_form, url: add_order_path(acting_user.cart(session_user)), html: {method: :put } do |f|
     = render partial: 'product_list', locals: {products: instruments, f: f, title_extra: daily_view_link }
     = render partial: 'product_list', locals: {products: services, f: f}
+    = render partial: 'product_list', locals: {products: timed_services, f: f}
```

Comment:
I initially noticed the `partial`/`local` and single quotes we want to get rid of, but let's save the linting here for another time in order to avoid obfuscating the diff.